Makefile

```diff
@@ -199,7 +203,7 @@ bundle: $(KUSTOMIZE) $(OPERATOR_SDK) generate
 		--select-optional name=operatorhub \
 		--optional-values=k8s-version=1.21 \
 		--select-optional suite=operatorframework
-	git diff --quiet -I'^    createdAt: ' bundle && git checkout bundle || true
+	git diff --no-ext-diff --quiet -I'^    createdAt: ' bundle && git checkout bundle || true
```

Comment:
Does this cause errors in any of our automations?

If not, I'm generally no a fan of local configurations/environments requiring changes in common tooling. If your difftool does not report an appropriate exit code, I'd suggest to start there.

Reply:
Honestly, this was mostly a quick fix to fix my own workflow and I understand that this _should_ usually not make it into the tooling, otherwise it'll get messy soon.

That said, now that I have looked through git docs and looked at how parameters are passed to the diff tool, I don't think `-I` is ever meant to interact properly with external tools. Both the `--quiet` and `-I` do not seem to be passed on to the external tool at all. So you could argue that `-I` is directly depending on the diff tool output and so you need to avoid running "unexpected tools".

I'm also open to just removing this parameter from the PR again, though.